src/main/java/frc/robot/commands/States/StoreFeeder.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.States;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.StateMachine.RobotState;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.StateMachine;
+import frc.robot.subsystems.Transfer;
+
+public class StoreFeeder extends Command {
+  StateMachine subStateMachine;
+  Intake subIntake;
+  Transfer subTransfer;
+
+  /** Creates a new StoreTransfer. */
+  public StoreFeeder(StateMachine subStateMachine, Intake subIntake, Transfer subTransfer) {
+    this.subStateMachine = subStateMachine;
+    this.subIntake = subIntake;
+    this.subTransfer = subTransfer;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subStateMachine);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    subStateMachine.setRobotState(RobotState.STORE_FEEDER);
+    subIntake.setIntakeRollerSpeed(Units.Percent.of(0));
+    subTransfer.setFeederSpeed(Units.Percent.of(0));
```

Comment:
```suggestion
    subIntake.setIntakeRollerSpeed(Units.Percent.zero());
    subTransfer.setFeederSpeed(Units.Percent.zero());
```
Units have special .zero() and .one() methods that we can use. .zero() will be nice for zeroing thing out since it is very explicit and easy to read. We don't strictly need to do this though